recipes/mesonlsp/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+# Fix problem with libarchive pkgconfig
+sed -i '/Requires.private/d' ${PREFIX}/lib/pkgconfig/libarchive.pc
+
+# Backport fixes for macOS. Remove with next release
+sed -i "s/'-Wl,-ld_classic'//" meson.build
+sed -i 's/and !defined(__x86_64__)//g' src/polyfill/polyfill.hpp
+
+export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+if [[ ${target_platform} =~ .*linux.* ]]; then
+    export CFLAGS="${CFLAGS} -pthread"
+    export CXXFLAGS="${CXXFLAGS} -pthread"
+fi
+
+meson --prefix=${PREFIX} \
```

Comment:
Probably this should allow you to omit the `--libdir` and will make sure that cross-compiling for the ARM arches works smoothly.
```suggestion
meson $MESON_ARGS --prefix=${PREFIX} \
```

Reply:
It turns out `${MESON_ARGS}` also provides `--prefix=${PREFIX}` and `--buildtype=release` so I was able to remove those as well.